golang/day_trader/src/main.go

```diff
@@ -308,15 +322,15 @@ func (s *server) DumpLog(ctx context.Context, req *pb.Command) (*pb.Response, er
 }
 
 func (s *server) DisplaySummary(ctx context.Context, req *pb.Command) (*pb.SummaryResponse, error) {
-	conn, err := grpc.Dial(logUrl, grpc.WithInsecure())
-	if err != nil {
-		log.Printf("Failed to dial to %s with %v", logUrl, err)
-	}
-	client := pb.NewLoggerClient(conn)
-	_, err = client.DisplaySummary(ctx, req)
-	if err != nil {
-		log.Println("DisplaySummary call to log server failed: ", err)
-	}
+	// conn, err := grpc.Dial(logUrl, grpc.WithInsecure())
+	// if err != nil {
+	// 	log.Printf("Failed to dial to %s with %v", logUrl, err)
+	// }
+	// client := pb.NewLoggerClient(conn)
+	// _, err = client.DisplaySummary(ctx, req)
+	// if err != nil {
+	// 	log.Println("DisplaySummary call to log server failed: ", err)
+	// }
```

Comment:
Is this going to be implemented later on?

Reply:
Yeah, this stuff will be in my next change.